src/inmanta/data/__init__.py

```diff
@@ -473,11 +304,37 @@ def coalesce_to_min(self, value_reference: str) -> str:
                 return f"COALESCE({value_reference}, FALSE)"
             elif self.base_type == int:
                 return f"COALESCE({value_reference}, 0)"
```

Comment:
Should we coalesce to -1 to make `null` strictly lower than 0? Or are our positive ints strictly positive (never 0)?

Reply:
done